### PR TITLE
add member registration social media announcement copy

### PR DESCRIPTION
**Pull request by Marie-L**

### CODE REVIEW

File: announcements/global_council_election_2020/social_media_announcement.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Copy for social media announcement for registering to be a PyLadies member
+
+
+- **Headline:** We're expanding  !
+- **Body :**  PyLadies member registration is now OPEN !  If you're interested in getting involved and making a difference in the community, then we’d love to hear from you !
+- **Footer :** Click the link below for more info (_insert link_)
```
Comment (Mariatta) [inline announcements/global_council_election_2020/social_media_announcement.md]: Perhaps the link is to the blog post? It has links to all the forms
https://www.pyladies.com/blog/Announcement-for-Registering-PyLadies-Members/announcement-for-registering-pyladies-members/

Comment (Marie-L) [reply announcements/global_council_election_2020/social_media_announcement.md]: @Mariatta ok sure

**Comment by AbigailDogbe** [comment]
@lorenanicole, we are set to make announcements about registration but do not have access to the twitter handle. How do we go about it? Also do you mind handling that?

**Comment by Mariatta** [comment]
I've posted the tweet: https://twitter.com/pyladies/status/1236357624871845888?s=19

Thanks!

**Comment by Marie-L** [comment]
@Mariatta Thank you

**Pull request merged**